### PR TITLE
Add message divider for unread messages in secure transcript

### PR DESCRIPTION
**Pull request by igorkravchenko**
Add message divider to indicate the presence of unread message in secure conversations transcript screen.

MOB-1734

### CODE REVIEW

File: GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift [hunk 1]
```diff
@@ -430,23 +440,26 @@ extension SecureConversations {
...
+    }
+}
 
-        @discardableResult
-        private func validateMessage() -> Bool {
-            let canSendAttachments =
-            fileUploadListModel.failedUploads.isEmpty &&
-            fileUploadListModel.activeUploads.isEmpty && !fileUploadListModel.isLimitReached
+// MARK: Input validation
+extension SecureConversations.TranscriptModel {
```
Comment (igorkravchenko) [inline GliaWidgets/SecureConversations/SecureConversations.ChatWithTranscriptModel.swift:447]: Move validation here to address linter warning.

**Review by rasmustautsglia — APPROVED** [review]
Looks good :)

**Review by EgorovEI — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!squash

**Comment by igorkravchenko** [comment]
!merge

**Pull request merged**